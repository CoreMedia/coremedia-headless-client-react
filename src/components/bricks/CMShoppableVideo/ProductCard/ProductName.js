// @flow
import { P } from '../../../basic/Paragraph';

const ProductName = P.extend`
  font-weight: 400;
  padding-bottom: 10px;
`;

ProductName.displayName = 'ProductName';

export default ProductName;
