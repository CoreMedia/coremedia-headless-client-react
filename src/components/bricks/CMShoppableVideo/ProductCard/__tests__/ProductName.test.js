// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import ProductName from '../ProductName';

describe('ProductName Component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<ProductName>Content</ProductName>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
