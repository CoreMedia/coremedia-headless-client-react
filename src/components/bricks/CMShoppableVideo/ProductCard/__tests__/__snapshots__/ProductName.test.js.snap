// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductName Component should render correctly 1`] = `
.c0 {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
  font-family: 'Simplon Norm Light','Lucida Sans','Lucida Sans Unicode','Lucida Grande',Arial, Helvetica,sans-serif;
  font-weight: 400;
  line-height: 1.4;
  font-size: 1em;
  font-weight: 400;
  padding-bottom: 10px;
}

@media (min-width:48em) {
  .c0 {
    font-size: 1em;
  }
}

@media (min-width:62em) {
  .c0 {
    font-size: 1.1em;
  }
}

@media (min-width:80em) {
  .c0 {
    font-size: 1.2em;
  }
}

<p
  className="c0"
>
  Content
</p>
`;
